code/modules/mob/mob.dm

```diff
@@ -31,8 +31,19 @@
 	else
 		living_mob_list += src
 	prepare_huds()
+	initialize_emotes()
 	..()
 
+/mob/proc/initialize_emotes()
+	var/datum/emote/E = new /datum/emote()
+	for(var/ET in emote_types)
+		for(var/datum/emote/EE in E.emote_lists[ET])
```

Comment:
There is so much wrong with this proc. 
Refer to the thing I said above about globals but even then ---
Why the fuck don't you just iterate over `subtypesof` and use the `initial()` trick to access the variables on each emote? 
Randomly generating a /datum/emote every time a mob joins purely for the sake of accessing a static list is fucking insane

Reply:
Also this means that it's a pain in the ass to make a mob able to use any emotes not hardcoded to their typepath

Reply:
You technically don't have to make a new emote. You can just do var/datum/emote/E. Let me see if that still works without the compiler warning...

Reply:
Okay, fixed! Also you can easily snowflake in emotes that aren't in the mob's emote_types, by initializing `emote_list` with emotes (and they'll take precedence over the "automatic" emotes). I don't see this being needed though.